cmd/agent/install_mac_os.sh

```diff
@@ -455,6 +468,15 @@ else
     $sudo_cmd launchctl kickstart "system/$service_name"
 fi
 
+# Set up and start the system-probe service if this version includes support for it
+if [ -f "${etc_dir}/com.datadoghq.sysprobe.plist.example" ]; then
```

Comment:
Super nit question: should we use the `sysprobe_service_name` variable there instead? (same question for two lines after)

Reply:
We definitely could, this was just based on https://github.com/DataDog/datadog-agent/blob/a5b59ed4f7ad2e77227af8ef2766691e8783792e/omnibus/package-scripts/agent-dmg/postinst#L103

This could definitely be better set as a variable too as an improvement.